cms/templates/js/course_grade_policy.underscore

```diff
@@ -2,31 +2,31 @@
   <div class="field text" id="field-course-grading-assignment-name">
   	<label for="course-grading-assignment-name"><%= gettext("Assignment Type Name") %></label>
   	<input type="text" class="long" id="course-grading-assignment-name" value="<%= model.get('type') %>" />
-  	<span class="tip tip-stacked"><%= gettext("e.g. Homework, Midterm Exams") %></span>
+  	<span class="tip tip-stacked"><%= gettext("The general category for this type of assignment, such as Homework or Midterm Exams. This name is visible to learners.") %></span>
```

Comment:
Thanks @pdesjardins Is there a reason why "for example" is preferred over "such as"?